samples/subsys/ipc/rpmsg_service/README.rst

```diff
@@ -36,6 +36,14 @@ Building the application for v2m_musca_b1
    :board: v2m_musca_b1
    :goals: debug
 
+Building the application for stm32h747i_disco_m7
+************************************************
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/ipc/rpmsg_service
+   :board: stm32h747i_disco_m7
+   :goals: debug
```

Comment:
I wonder why the 'goals' is debug here (aside from copy/paste).
Besides, shouldn't we document m4 target should be build/flashed as well ?

Reply:
openamp and rpmsg_service sample build m7 and m4 target at same time. 
m7 at 
```
build
```
m4 at
```
build/openamp_remote-prefix/src/openamp_remote-build
```

Reply:
I will try to adjust stm32h747 board to fully match it.

Reply:
'zephyr-app-commands' in README.rst not support add --build-dir option for west flash only. ':build-dir:' will make west build and flash both add a "--build-dir" option.
The nested remote project in openamp sample, makes it hard to use 'zephyr-app-commands' for stm32h747i_disco_m4. 
I think there have two way can fix it. make 'zephyr-app-commands' more flexible; or make a big change for this sample.

Reply:
Right, I've came across this as well on [this board](https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/b_u585i_iot02a). I ended up using `   .. code-block:: console`.
Don't hesitate to make use of it as that's important for people to easily reproduce

Reply:
Having had a try, I confirm that the README should be fixed as the build procedure is unusual.
At least you should mention:
- running the _m7 build command also buid the _m4
- flashing procedures for both cores

Also, it could be interesting to add a note stating that to get the console working on m4 side you need to disable console on m7 and enable on m4. That is not handy but the fastest way to get console on m4 and that's not obvious for most users I guess. (Ideally, it would be nice having usb based console on m4 enabled by default but that's another topic).

Reply:
@erwango Thanks!